README.rst

```diff
@@ -0,0 +1,38 @@
+Tutor Plugin Indexes
+====================
+
+Repository with the collection of Tutor Plugins used for edunext in their installations.
+
+Usage
+-----
+
+To use this index in your instance you can follow the instructions in the `official Tutor Documentation`_.
+
+Add the index with:
+
+.. code-block:: bash
+  
+  tutor plugins index add https://raw.githubusercontent.com/eduNEXT/tutor-plugin-indexes/main/
+  
+Remove the index with:
+
+.. code-block:: bash
+  
+  tutor plugins index remove https://raw.githubusercontent.com/eduNEXT/tutor-plugin-indexes/main/
+  
+
+If you need to update the cache, use the command ``tutor plugins update``.

```

Comment:
I'm pretty new to indexes, so could you explain a bit why I would need to do this?

Reply:
Sure, I hope it's more clear now